installing/installing_aws/installing-aws-wavelength.adoc

```diff
@@ -0,0 +1,280 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+
+:context: installing-aws-wavelength
+
+// Context Variables for edge zones
+:aws-service-name-full: AWS Wavelength
+:aws-service-name-short: Wavelength Zones
+:aws-service-name-single: Wavelength Zone
+:zone-type: wavelength-zone
+
+[id="installing-aws-wavelength"]
+= Installing a cluster on AWS with worker nodes on {aws-service-name-full}
+
+// ATTENTION: All the content from here is shared between the pages:
+// installing-aws-local-zones
+// installing-aws-wavelength
+
+ifeval::["{zone-type}" == "{aws-zone-type-local}"]
+:aws-localzones:
+:aws-doc-zone: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-local-zones
+endif::[]
+ifeval::["{zone-type}" == "{aws-zone-type-wavelength}"]
+:aws-wavelength:
+:aws-doc-zone: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-wavelength-zones
+endif::[]
+
+toc::[]
+
+// Intro Installing in AWS Edge zones ({aws-service-name-full})
+include::modules/aws-edge-zones-installing-intro.adoc[leveloffset=+0]
+
+// Additional Permissions for {aws-service-name-full}
+include::modules/aws-edge-zones-permissions.adoc[leveloffset=+0]
+
+== About {aws-service-name-full} and edge compute pool
+
+// Cluster limitations in {aws-service-name-full}
+include::modules/aws-edge-zones-cluster-limitations.adoc[leveloffset=+2]
+
+// Edge compute pools and AWS {aws-service-name-full}
+include::modules/edge-machine-pools-aws-local-zones.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../networking/changing-cluster-network-mtu.adoc#mtu-value-selection_changing-cluster-network-mtu[Changing the MTU for the cluster network]
+* xref:../../networking/changing-cluster-network-mtu.adoc#nw-ovn-ipsec-enable_configuring-ipsec-ovn[Enabling IPsec encryption]
+* xref:../../nodes/scheduling/nodes-scheduler-taints-tolerations.adoc#nodes-scheduler-taints-tolerations-about_nodes-scheduler-taints-tolerations[Understanding taints and tolerations]
+
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../storage/understanding-persistent-storage.adoc#pvc-storage-class_understanding-persistent-storage[Storage classes]
+
+* xref:../../networking/ingress-sharding.html#nw-ingress-sharding_ingress-sharding[Ingress Controller sharding]
+
+== Prerequisites for the installation
+
+// Opting in to the {aws-service-name-short} Group
+include::modules/installation-aws-add-local-zone-locations.adoc[leveloffset=+2]
+
+// Internet access for OpenShift Container Platform
+include::modules/cluster-entitlements.adoc[leveloffset=+2]
+
+// Obtaining an AWS Marketplace image
```

Comment:
In your preview for WZ, only one step is available. The same module steps for LZ should apply to WZ docs for this module?

Reply:
> In your preview for WZ, only one step is available. 

could you please elaborate that part?

Reply:
NP. Your PR's preview for the [Obtaining an AWS Marketing Image](https://69128--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-wavelength-zones#installation-aws-marketplace-subscribe_installing-aws-wavelength-zone) for WZ only has one step, while the LZ doc has more steps. Should the section in oth docs be identical?

![Screenshot from 2023-12-22 11-23-09](https://github.com/openshift/openshift-docs/assets/57954076/4cdc45c6-f0a4-45c4-87e8-49dd5c6e0173)

Reply:
probably it was removed by mistake, the intention wasn't touch/remove that part.